Source/WebKit/WebProcess/GPU/media/SourceBufferPrivateRemote.h

```diff
@@ -149,8 +148,9 @@ class SourceBufferPrivateRemote final
 
     std::atomic<uint64_t> m_totalTrackBufferSizeInBytes = { 0 };
 
-    bool isGPURunning() const { return !m_disconnected; }
-    bool m_disconnected { false };
+    bool isGPURunning() const { return !m_shutdown && !m_removed; }
+    std::atomic<bool> m_shutdown { false };
```

Comment:
m_isShuttingDown?

Reply:
the GPU Process is shutting down, once this method is finished, it would be shutdown.

so m_isShutdown would be more suitable; however `m_shutdown` is more used in the tree than `m_isShutdown` and is what was used in MediaSourcePrivateRemote